docs: update how-tos with roles (#687)
juju/terraform-provider-juju: docs-rtd/howto/manage-groups.md

(manage-access-to-a-group)=
## Manage access to a group

> When using Juju with JAAS, to grant one or more users, service accounts, roles, and/or groups access to a group, in your Terraform plan add a resource type `juju_jaas_access_group`, specifying the group ID, the JAAS group access level, and the list of desired users, service accounts, roles, and/or groups. For example:

Reviewer: Tried to break it down based on my last comment.

Somewhere at the top of the page, mention that group access controller is only possible when using Juju with JAAS.

```suggestion
To grant access to a group, use the resource type `juju_jaas_access_group` in your Terraform plan. Access can be granted to one or more users, service accounts, roles, and/or groups. The resource must include the group ID, the JAAS group access level, and the list of desired users, service accounts, roles, and/or groups. For example:
```
